temperature/models.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy import Column, Integer, DateTime, Float, ForeignKey
+from sqlalchemy.orm import relationship
+
+from city.models import City
+from database import Base
+
+
+class Temperature(Base):
+    __tablename__ = "temperature"
+
+    id: int = Column(Integer, primary_key=True, index=True)
+    date_time = Column(DateTime, nullable=False)
+    temperature = Column(Float, nullable=False)
+    city_id = Column(Integer, ForeignKey("city.id"))
```

Comment:
The `city_id` column is defined with a foreign key reference to `city.id`. Ensure that the `City` model has a primary key defined as `id` to maintain referential integrity.